metadata/config/service.json

```diff
@@ -0,0 +1,12 @@
+{
+  "_id": "service",
+  "_type": "config.service",
+  "dataRetentionDuration": 28,
+  "homepageUrl": "https://www.gov.uk",
+  "name": "Service name goes here",
+  "phase": "alpha",
+  "phaseText": "Service phase text goes here",
+  "serviceEmailAddress": "form-builder.no-reply@digital.justice.gov.uk",
```

Comment:
What's the `.no-reply` bit about? Is that just for the form user to see and understand they can't reply to the email?

Reply:
I've just reverted it to be our standard form-builder@digital.justice.gov.uk address in the absence of us actually having any idea what the email should be (or could be)